src/components/backend-ai-storage-list.ts

```diff
@@ -147,7 +147,11 @@ export default class BackendAiStorageList extends BackendAIPage {
     MB: Math.pow(10, 6),
     GB: Math.pow(10, 9),
     TB: Math.pow(10, 12),
-    PB: Math.pow(10, 15)
+    PB: Math.pow(10, 15),
```

Comment:
It seems this change is unrelated to this PR (maybe inserted into this accidentally)

Reply:
It will help bit-wise quota calculation, I guess.